src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -3069,6 +3070,10 @@ public sealed partial class PortablePdbBuilder
         public void UIntPtr() { }
         public void VoidPointer() { }
     }
+    public readonly struct SwitchInstructionEncoder
```

Comment:
I see we went with this instead of `InstructionEncoder` (makes sense; this is the approved API)